from unittest import TestCase


class TestData(TestCase):
    def test_error(self):
        self.fail()

    def test_exit(self):
        self.fail()
